src/app/beer_garden/app.py

```diff
@@ -82,11 +82,16 @@ def initialize(self):
         ]
 
         # Only want to run the MongoPruner if it would do anything
+        cancel_after = config.get("db.ttl.in_progress")
```

Comment:
I think I commented about not doing the `config.get` twice in a row here.  Just get the "db.ttl" data once, then use it in the two places you need it.